scripts/build-depends.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python3
+from email.parser import Parser
+with open('debian/control') as fp:
```

Comment:
would you wrap this code in a `if __name__ == '__main__':` block?

Reply:
Done!